src/App.js

```diff
@@ -1,15 +1,55 @@
 import './App.css';
-import Header from './components/Header';
+import NavaMenu from './components/NavaMenu';
 import BodyRouter from './components/BodyRouter';
 import React, { useState } from 'react'
 import { initializeApp } from "firebase/app";
 import { getFirestore } from 'firebase/firestore';
-import { ThemeProvider } from '@mui/material/styles';
 import CssBaseline from "@mui/material/CssBaseline";
-import muiTheme from './styles/MuiTheme';
 import AppContext from './components/AppContext';
 import config from './config';
 
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+const theme = createTheme({
```

Comment:
Why does this belong here vs. in it's own file?

Reply:
I was having trouble with MuiOverrides, docos that I found through stack often said to internalize the theme in the app function, didn't see any issue to it?